ooniprobe/Test/Suite/AbstractSuite.m

```diff
@@ -100,6 +100,13 @@ - (void)showLocalNotification {
 }
 
 -(NSArray*)getTestList {
+    if (self.autoRUn){
+        for (int i = 0; i < [self.testList count]; ++i) {
+            AbstractTest *current = self.testList[(NSUInteger) i];
+            current.autoRUn = YES;
+            self.testList[(NSUInteger) i] = current;
```

Comment:
```suggestion
            self.testList[(NSUInteger) i].autoRUn = YES;
```

A bunch of comments and questions:

1. beware of the capitalization: `RUn` should be `Run`
2. would the code above work?
3. what is the lifecycle of this `self.testList`?
4. what is the implication of setting `autoRun` to `YES`? could it stick to future runs?
5. is there a way to write this code to avoid mutating the content of `self`?

Reply:
this would not work. the direct assignment posses some problems with the types. i tried that.